### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by kiberlain** [retypeapp/retype#13] [Q&A] [answered]
1. How to replace the word "filter" in the search with another?
2. Where to edit styles?
3. Will it be possible to run your ssg on such hosting as netlify, vercel, hostman, render.com etc? (i mean the build process. For example, for react and docsify this works, but for retype it does not)

**Comment by fabriciomurta** [comment, marked as answer]
1. Currently there's no mean to replace that word; in the future we can support this; I suggest you create an issue specific to this, so we can log a feature request -- so other people wanting the same feature can post their feedback and we prioritize the issue for how many people want it.
2. Currently also there's no customization for styling; we are thinking on nice ways to quickly style up your docs, but first focusing on functionality -- the provided styling supports light and dark styling though.
3. I'm not sure what you mean with "ssg"
If you mean the GitHub action at [retypeapp/action-build](https://github.com/retypeapp/action-build); we have it leave you freedom to extend the next steps to publish it wherever you want from your github action.
Currently we have [retypeapp/action-build](https://github.com/retypeapp/action-build) to simply build the docs website; then [retypeapp/action-github-pages](https://github.com/retypeapp/action-github-pages) to push the built documentation to a branch/directory where you set up your GitHub Pages on the repository. Based on the latter, you can create any action to push the built website to any hosting you might have be it via FTP SFTP, SSH, and so on.

**Comment by kiberlain** [comment]
ssg - static site generator

**Comment by fabriciomurta** [comment]
Hey, we've moved your issue into a thread, so don't mind the "issue closed" message; we just converted it into a discussion.
